dev/io.openliberty.org.eclipse.microprofile.health.javax/bnd.bnd

```diff
@@ -0,0 +1,31 @@
+#*******************************************************************************
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion: 1.0
+
+-sub: health.4.0.javax.bnd
```

Comment:
Not sure why you did a sub bnd file here.  You should have been able to just keep it all in the main bnd.bnd file.

Reply:
Ah, sorry. This should've been `*.bnd`.
Set up this way in anticipation for `mpHealth-4.1` which would probably also require backporting and _javaxifying_.